build/gh-release.js

```diff
@@ -36,9 +36,8 @@ if (args.prerelease || npmargs.some(function(arg) { return /next/.test(arg); }))
 ghrelease(options, function(err, result) {
   if (err) {
     console.log('Unable to publish release to github');
-    console.log(err);
+    console.log('err:', err, '\nresult:', result);
```

Comment:
Maybe we should switch this to `console.error` not that it really matters that much. It also seems weird to have an explicit new line here. We should probably just use another `console` statement.